examples/rest-api-client-demo/package.json

```
@@ -13,6 +13,8 @@
     "directory": "examples/rest-api-client-demo"
   },
   "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "tsc --build",
```

```suggestion
    "build": "tsc --build --force",
```

Does need `--force` option?